File: work/nh/projects/nh_it_apps/api/Models/Entities/AppService.cs
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace api.Models.Entities
{
    public class AppService : BaseEntity
    {
        [Required]
        public int NhAppId { get; set; }
        
        [Column(TypeName="varchar(50)")]
        public string Name { get; set; }
        
        [Column(TypeName="varchar(20)")]
        public string Type { get; set; }
        
        public IEnumerable<Server> Servers { get; set; }
    }
}

File: work/nh/projects/nh_it_apps/client/src/app/models/AppService.ts
import { Server } from "./Server";

export class AppService {
    id: number = 0;
    nhAppId : number = 0;
    name: string = "";
    type: string = "";
    comments: string = "";
    lastUpdatedBy: string = "";
    lastUpdated: Date = new Date();
    servers: Server[] = [];
}
